src/pages/search.js

```diff
@@ -0,0 +1,12 @@
+import Searchbar from "@components/Searchbar/Searchbar";
+import React, { useEffect, useState } from "react";
```

Comment:
```suggestion
import { useEffect, useState } from "react";
```

you do not need to import React here